### PR TITLE
Seafood City

### PR DESCRIPTION
**Pull request by CloCkWeRX**
Fix https://github.com/alltheplaces/alltheplaces/issues/8051

### CODE REVIEW

File: locations/spiders/seafood_city.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+class SeafoodCitySpider(StoreLocatorPlusSelfSpider):
+    name = "seafood_city"
+    item_attributes = {
+        "brand_wikidata": "Q7440359",
+        "brand": "Seafood City",
+    }
+    max_results = 100
+
+    def start_requests(self):
+        url = f"https://www.seafoodcity.com/wp-admin/admin-ajax.php"
```
Comment (CloCkWeRX) [inline locations/spiders/seafood_city.py:15]:
```suggestion
        url = "https://www.seafoodcity.com/wp-admin/admin-ajax.php"
```

**Comment by iandees** [comment]
It looks like the `street_address` should move to the `full_address`

File: locations/spiders/seafood_city.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    item_attributes = {
+        "brand_wikidata": "Q7440359",
+        "brand": "Seafood City",
+    }
+    max_results = 100
+
+    def start_requests(self):
+        url = "https://www.seafoodcity.com/wp-admin/admin-ajax.php"
+        lat = 37.09024
+        lon = -95.712891
```
[the remark this answers is not in the document]
Comment (CloCkWeRX) [reply locations/spiders/seafood_city.py:17]: Darn, had assumed they had opened it all up.

[the remark this answers is not in the document]
Comment (CloCkWeRX) [reply locations/spiders/seafood_city.py:17]: Okay, this is a bit weird;

With
```
from scrapy.http import FormRequest

from locations.storefinders.store_locator_plus_self import StoreLocatorPlusSelfSpider


class SeafoodCitySpider(StoreLocatorPlusSelfSpider):
    name = "seafood_city"
    item_attributes = {
        "brand_wikidata": "Q7440359",
        "brand": "Seafood City",
    }
    allowed_domains = ['www.seafoodcity.com']
    searchable_points_files = ["us_centroids_iseadgg_458km_radius.csv"]
    search_radius = 500
    max_results = 25

```
I only get
```
2024-04-01 02:10:11 [locations.pipelines.duplicates] INFO: Dropped 48 duplicate items
2024-04-01 02:10:11 [scrapy.statscollectors] INFO: Dumping Scrapy stats:
{'atp/brand/Seafood City': 25,
 'atp/brand_wikidata/Q7440359': 25,
 'atp/category/shop/supermarket': 25,
 'atp/field/country/from_reverse_geocoding': 21,
 'atp/field/email/missing': 25,
 'atp/field/image/missing': 25,
 'atp/field/opening_hours/missing': 25,
 'atp/field/operator/missing': 25,
 'atp/field/operator_wikidata/missing': 25,
 'atp/field/phone/missing': 25,
 'atp/field/postcode/missing': 25,
 'atp/field/state/from_reverse_geocoding': 4,
 'atp/field/twitter/missing': 25,
 'atp/field/website/missing': 25,
 'atp/nsi/perfect_match': 25,
 'downloader/request_bytes': 19613,
 'downloader/request_count': 43,
 'downloader/request_method_count/GET': 1,
 'downloader/request_method_count/POST': 42,
 'downloader/response_bytes': 59439,
 'downloader/response_count': 43,
 'downloader/response_status_count/200': 43,
 'elapsed_time_seconds': 55.625663,
 'finish_reason': 'finished',
 'finish_time': datetime.datetime(2024, 4, 1, 2, 10, 11, 297734, tzinfo=datetime.timezone.utc),
 'httpcompression/response_bytes': 122149,
 'httpcompression/response_count': 43,
 'item_dropped_count': 48,
 'item_dropped_reasons_count/DropItem': 48,
 'item_scraped_count': 25,
 'log_count/DEBUG': 127,
 'log_count/INFO': 9,
 'memusage/max': 159010816,
 'memusage/startup': 159010816,
 'response_received_count': 43,
 'robotstxt/request_count': 1,
 'robotstxt/response_count': 1,
 'robotstxt/response_status_count/200': 1,
 'scheduler/dequeued': 42,
 'scheduler/dequeued/memory': 42,
 'scheduler/enqueued': 42,
 'scheduler/enqueued/memory': 42,
 'start_time': datetime.datetime(2024, 4, 1, 2, 9, 15, 672071, tzinfo=datetime.timezone.utc)}
2024-04-01 02:10:11 [scrapy.core.engine] INFO: Spider closed (finished)
ubuntu@codespaces-83b177:/workspaces/alltheplaces$ 
```

**Comment by CloCkWeRX** [comment]
Closing for now, I am not likely to have time to improve this.

**Pull request closed without merging**